ocs_samples/basic_samples/OmfIngress/DotNet/readme.md

```diff
@@ -0,0 +1,173 @@
+.NET Samples 
+============
+
+Building a Client with the Ingress Client Libraries
+---------------------------------------------------
+
+The sample described in this section makes use of the OSIsoft Ingress Client Libraries. When working in .NET, 
+it is recommended that you use these libraries. The libraries are available as NuGet packages. The packages used are:
+
+* OSIsoft.Omf
+* OSIsoft.OmfIngress
+* OSIsoft.Identity.AuthenticationHandler
+
+The libraries offer a framework of classes that make client development easier.
+
+Configure constants for connecting and authentication
+-----------------------------------------------------
+
+The OMF Ingress Service is secured by obtaining tokens from the Identity Server. Such clients 
+provide a client application identifier and an associated secret (or key) that are 
+authenticated against the server. The sample includes an appsettings.json configuration 
+file to hold configuration strings, including the authentication strings. You must 
+replace the placeholders with the authentication-related values you received from OSIsoft. The application requires two Client Credential Clients, one to manage OMF Ingress connetions and one to send data from a mock device. For information on how to obtain these client IDs and secrets, see the [Client Credential Client Documentation](https://ocs-docs.osisoft.com/Documentation/Identity/Identity_ClientCredentialClient.html).
+
+```
+	{
+        "TenantId": "REPLACE_WITH_YOUR_TENANT_ID",
+        "NamespaceId": "REPLACE_WITH_YOUR_NAMESPACE_ID",
+        "Address": "https://dat-b.osisoft.com",
```

Comment:
Is there a chance for confusion of someone entering in their OMF URL here from the Connections page?  The one with tenantID and namespace in there.  Maybe a note above just in case someone is